File: src/main/java/com/mymedia/nagasu/dto/VideoDetailsResponse.java
package com.mymedia.nagasu.dto;

import java.util.List;

public record VideoDetailsResponse(
        String id,
        String name,
        String title,
        String artist,
        List<String> tags,
        String description,
        String thumbnailUrl,
        String videoUrl,
        String videoSubtitleUrl,
        int viewCount
) {
}

File: src/main/kotlin/com/mymedia/nagasu/dto/VideoDetailsResponse.kt
package com.mymedia.nagasu.dto

data class VideoDetailsResponse(
    val id: String,
    val name: String,
    val title: String,
    val artist: String,
    val tags: List<String>,
    val description: String,
    val thumbnailUrl: String,
    val videoUrl: String,
    val videoSubtitleUrl: String?,
    val viewCount: Int
)
